### ISSUE TITLE
Error responses have incorrect codes

### ISSUE THREAD

**Issue by randomjunk** [etspaceman/kinesis-mock#105]
The error responses are prefixed with `kinesis.mock.` which makes them incompatible with expected error codes in at least some AWS SDKs. This means we can't replace kinesalite in our tests without code modifications, which shouldn't be necessary here.

eg, we have the following golang code (following the [suggested error handling guide](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/handling-errors.html)):
```go
    _, err := k.DescribeStream(&kinesis.DescribeStreamInput{
        StreamName: aws.String(streamName),
    })

    if err != nil {
        if awsErr, ok := err.(awserr.Error); ok {
            switch awsErr.Code() {
            case kinesis.ErrCodeResourceNotFoundException:
                // create stream

            default:
                log.Printf("Unknown error: "+awsErr.Code())
            }
        }
    }
```

which is outputting:
```
Unknown error: kinesis.mock.ResourceNotFoundException
```

This seems to be an issue [here](https://github.com/etspaceman/kinesis-mock/blob/da9b5b28b7a25ec6301f17aec7746acb70f6f4de/src/main/scala/kinesis/mock/KinesisMockRoutes.scala#L457) -- not done Java for a while, but I believe just replacing with `getSimpleName` would fix.

Thanks.

**Comment by etspaceman** [comment]
Hey thanks for your interest. Good catch. I can update this.

**Comment by etspaceman** [comment]
Fixed with #106
